lib/avo/services/authorization_service.rb

```diff
@@ -5,13 +5,13 @@ class AuthorizationService
       attr_accessor :record
 
       class << self
-        def authorize(user, record, action, **args)
+        def authorize(user, record, action, policy_class: nil, **args)
           return true if skip_authorization
           return true if user.nil?
 
```

Comment:
Maybe we should add this here too as a safeguard?

```suggestion
          policy_class ||= Pundit.policy(user, record)&.class

```